client/src/components/PinnedFAQ/PinnedFAQ.js

```diff
@@ -0,0 +1,41 @@
+import _ from "lodash";
+import React from "react";
+
+import { KeyConceptList } from "src/components/KeyConceptList/KeyConceptList";
+
+import { Gov } from "src/models/organizational_entities";
+
+import { secondaryColor } from "src/style_constants/colors.interop.scss";
+
+import { SomeThingsToKeepInMind } from "./faq_utils";
+
+export class PinnedFAQ extends React.Component {
+  render() {
+    const {
+      q_a_key_pairs,
+      TM,
+      is_initially_expanded,
+      background_color,
+      subject,
```

Comment:
keeping the subject props for now to move this PR along

Reply:
Ok, I'll open the follow up issue or knock it out my self once this merges